main/libmicrohttpd/APKBUILD

```diff
@@ -2,15 +2,15 @@
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=libmicrohttpd
-pkgver=0.9.62
+pkgver=0.9.63
 pkgrel=0
 pkgdesc="A small C library that is supposed to make it easy to run an HTTP server as part of another application."
 url="https://www.gnu.org/software/libmicrohttpd/"
 arch="all"
-license="GPL"
+license="LGPL-2.1-or-later"
 makedepends="curl-dev libgcrypt-dev gnutls-dev"
 subpackages="$pkgname-dev $pkgname-doc"
-source="ftp://ftp.gnu.org/gnu/libmicrohttpd/$pkgname-$pkgver.tar.gz"
+source="https://ftpmirror.gnu.org/gnu/libmicrohttpd/$pkgname-$pkgver.tar.gz"
```

Comment:
We generally avoid fetching from mirror services, please revert.

Reply:
Reverted. The main host was offlice so I have switched to mirror service. Changed the protocol from ftp to https.

Reply:
If you want to change the `source=` URL from `ftp -> https` you need to do one of the following:

- reword the commit message to: `main/libmicrohttpd: upgrade to 0.9.63, switch to https`
- create a separate commit under the same PR with the message `main/libmicrohttpd: switch to https`. This commit will only include your edit to the `source=` URL
- revert the `source=` edit because the [builders can handle](https://dev.alpinelinux.org/buildlogs/build-edge-x86_64/main/libmicrohttpd/libmicrohttpd-0.9.62-r0.log) `ftp`

Reply:
Opted for option 1.